specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -201,7 +205,8 @@
             "$ref": "./examples/ContainerGroupsDelete.json"
           }
         },
-        "description": "Delete container groups.",
+        "summary": "Delete the specified container groups.",
```

Comment:
we only support delete on single container group, do we need the plural form groups?

Reply:
Agree, updating the description. However why are all the operationIds plural (ContainerGroups_Delete)? I suppose the plural form is the default ARM resource name is why?